src/controllers/user.controller.js

```diff
@@ -0,0 +1,26 @@
+import userService from "../services/user.service"
+
+const { retrieveUser, upDateUserInfo } = userService
+
+const getUserInfo = async (req, res) => {
+    const userInfo = await retrieveUser(req.params.userId)
+    if ( userInfo != "" ) {
+        res.status(200).send({"requested user" : userInfo})
+    } else {
+        res.status(404).send({"message": "No user with such provided information"})
+    }
+
+}
+
+const upDateUser = (req, res) => {
+    const dbResponse = upDateUserInfo({user:req.body}, req.params.userId)
+    if (dbResponse === 'true') {
+        res.status(200).send({"message" : "Profile information updated successfully"})
+    } else if (dbResponse === "false") {
+        res.status(400).send({"message" : "Unable to update your profile"})
+    }
+}
+
+
+
+export default { getUserInfo, upDateUser };
```

Comment:
Can you transform your functions into a class?
For readability and solid principles